connectivity/drivers/nfc/TARGET_M24SR/include/nfc/m24sr_driver.h

```diff
@@ -0,0 +1,1572 @@
+/**
+ ******************************************************************************
+ * @file    m24sr_driver.h
+ * @author  ST Central Labs
+ * @brief   This file provides a set of functions to interface with the M24SR
+ *          device.
+ ******************************************************************************
+ * @attention
+ *
+ * <h2><center>&copy; COPYRIGHT(c) 2018 STMicroelectronics</center></h2>
```

Comment:
This will fail Mbed OS CI unless it has SPDX identifiers in it.

Reply:
@harmut01 You recently worked on licensing validation in CI. What should we do about partners source code missing SPDX?

@jeromecoutant @apalmieriGH  what would you advise here?

Reply:
Although we don't strictly enforce it at the moment, all code should have an SPDX identifier. This is from our contributors guidelines: 

> In all cases, whatever license you use, please use an SPDX license identifier in every source file following the recommendations to make it easier for users to understand and review licenses.

https://os.mbed.com/docs/mbed-os/v6.6/contributing/license.html